notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2224 @@
+{
```

Comment:
<p>Remove all the</p><blockquote>For more details, please check&nbsp;<a href="https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.17/google_cloud_pipeline_components.experimental.evaluation.html#google_cloud_pipeline_components.experimental.evaluation.GetVertexModelOp" target="_blank">here</a>.</blockquote><p>Instead, add this line at the end of this text cell: </p><p>Learn more about [Google Cloud Pipeline Model Evaluation components](https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.20/google_cloud_pipeline_components.experimental.evaluation.html).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Reply:
<p>done</p>